upgrade.adoc

```diff
@@ -0,0 +1,494 @@
+:product-name: Zimbra Collaboration
+:product-edition-commercial: Network Edition
+:product-short: Zimbra
+:product-family: Zimbra
+:product-provider: Zimbra
+:product-abbrev: ZCS
+:product-version: 8.8.0 GA
+:product-admin-guide: {product-name} Administration Guide
+:icons: font
+:title-logo-image: images/zimbra-logo-non-interlaced.png
+:showlinks:
+:source-highlighter: coderay
+:toc: left
+
+= {product-name} {product-version} Upgrade Steps
+
+== Before you upgrade
+
+The following tasks might need to be performed before you
+upgrade. After you review the tasks in this section, go to Upgrade
+Instructions. Be sure to read the
+https://wiki.zimbra.com/index.php?title=Zimbra_Releases/8.7.0[release note] information before upgrading.
+
+[NOTE]
+====
+https://bugzilla.zimbra.com/show_bug.cgi?id=105056[Bug
+105056]
+noted a problem that can occur during a rolling upgrade if two
+factor authentication (2FA) is enabled before all mailbox servers have
+been upgraded to 8.7.  In particular, pre-8.7 mailbox servers are not
+compatible with 2FA.
+
+Accordingly, it is recommended that 2FA is not enabled until all
+mailbox servers have been upgraded to 8.7.
+====
+
+=== Database Integrity Checking
+
+Some customers have had corrupted databases prior to upgrade, and the
+upgrade has in some of those cases exacerbated the problem. In order
+to detect any corrupted databases as early as possible, we have added
+an optional step to check the MariaDB database with
+zmdbintegrityreport prior to making any system changes. You are
+prompted to decide if you would like to run the zmdbintegrityreport.
```

Comment:
Perhaps `zmdbintegrityreport`?
Does the terminology `The Zimbra Collaboration` and here, `the zmdbintegrityreport` sound really weird to others like it does to me?  I would drop the "the" in both cases.